### ISSUE TITLE
PageSmith SQL Update Incompatible with Postgresql

### ISSUE THREAD

**Issue by jlbooker** [AppStateESS/phpwebsite#249]
```
"SQLSTATE[42601]: Syntax error: 7 ERROR: syntax error at or near "CHANGE" LINE 1: ALTER TABLE "ps_page" CHANGE "show_title" "hide_title" small... 
```

Seems to be in https://github.com/AppStateESS/phpwebsite/blob/master/mod/pagesmith/boost/update.php#L441

Postgresql doesn't support the `CHANGE` syntax. The Postgresql syntax is `ALTER TABLE name RENAME column TO new_column`.

**Comment by jlbooker** [comment]
To update manually
```
alter table ps_page rename show_title to hide_title;
alter table ps_page add column deleted smallint default 0;
```

**Comment by Stardog** [comment]
I ran into this too.

See this commit: beb576ca4d62dcbb37c912bfa33298638e7d7817

The Table classes are database specific thus you can run rename and alter on them and it will use the correct syntax. I am not sure why I added those methods to the Datatype.

**Comment by jlbooker** [comment]
It's currently preventing PageSmith from being upgraded on any site running on Postgresql. We can either fix the syntax, or just run the above sql by hand and bump the version number manually so that Boost can finish the upgrades.

**Comment by Stardog** [comment]
Should be fixed by ad32e2b675ecce82b2e7f06fa5b70882c634b338
